requirements-dev.txt

```diff
@@ -1,4 +1,5 @@
 -rrequirements.txt
+black
```

Comment:
Do we want to pin the version here?

Reply:
I was torn about this myself, but since you suggested it, it has swayed me.